SimpleServlet/src/classes/RequestHandler.java

```diff
@@ -9,33 +9,96 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/**
- * Servlet implementation class HelloWorld
- */
+import com.mongodb.MongoClient;
+import com.mongodb.MongoClientURI;
+import com.mongodb.ServerAddress;
+
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.MongoCollection;
+
+import org.bson.Document;
+import java.util.Arrays;
+import com.mongodb.Block;
+
+import com.mongodb.client.MongoCursor;
+import static com.mongodb.client.model.Filters.*;
+import com.mongodb.client.result.DeleteResult;
+import static com.mongodb.client.model.Updates.*;
+import com.mongodb.client.result.UpdateResult;
+import java.util.ArrayList;
+import java.util.List;
+
+
 @WebServlet("/Example")//servlet url
 public class RequestHandler extends HttpServlet {
-	private static final long serialVersionUID = 1L;
-	
-	private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response) throws IOException {
-		response.setContentType("text/html");
-		try(PrintWriter printWriter = response.getWriter()){
-			printWriter.println(message);
-		}			
+	private static final long serialVersionUID = 1L;	
+
+	private void sendToMongoDB(String message, String operation, HttpServletRequest request, HttpServletResponse response) throws IOException {
+		
+		response.setContentType("text/html");		
+		try(PrintWriter printWriter = response.getWriter()){		
+			User user = new User();
+
+			user.id = Integer.parseInt(request.getParameter("id"));
+			user.firstName = request.getParameter("firstName");
+			user.middleName = request.getParameter("middleName");
+			user.lastName = request.getParameter("lastName");
+			
+			printWriter.println("<h1>First name:" + user.firstName + "</h1>" + 
+					"<h1>Midle name:" + user.middleName + "</h1>" + 
+					"<h1>Last name:" + user.lastName + "</h1>");				
+			
+			try(MongoClient mc = new MongoClient("localhost", 27017)){
+			
+				MongoDatabase database = mc.getDatabase("UserDB");
+				MongoCollection<Document> collection = database.getCollection("users");
+				
+				if(operation == "Create") {
+					collection.insertOne(new Document("id", user.id)
+											.append("firstName", user.firstName)
```

Comment:
Format you code, if you use Eclipse try use combination Ctrl+Shift+F. If you use Intellig IDEA try: Ctrl + Alt + L or Ctrl + Windows Key + Alt + L